CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 <!--
 Contains editorialized release notes. Raw release notes should go into `RELEASE-NOTES.txt`.
 -->
+## 20.1
+Explore smoother shopping experiences with our latest update! We've fixed an array of bugs, improved order creation flows, and enhanced campaign functionalities for non-Western Arabic numbering systems. Plus, enjoy streamlined order editing, updated campaign budgets, and a fresh barcode scanner icon. This version also resolves navigation crashes in the Hub Menu for iOS 18 beta users.
```

Comment:
> "and enhanced campaign functionalities for non-Western Arabic numbering systems".

For better clarity, I'd suggest rewriting this to:

"and added support for Blaze ads creation on devices with non-Western Arabic numbering system".

Reply:
Thanks! @joshheald added the suggestion to the changes